FastReport.Core.Web/Application/WebReportDesigner.cs

```diff
@@ -564,7 +564,7 @@ string GetPOSTReport(HttpContext context)
         {
             string requestString = "";
             using (TextReader textReader = new StreamReader(context.Request.Body))
-                requestString = textReader.ReadToEndAsync().Result;
+                requestString = textReader.ReadToEnd();
```

Comment:
Please, do not change this code, because in Core3 as default disabled streams sync.

Reply:
Good spot! Removed